.travis.yml

```diff
@@ -38,7 +38,7 @@ before_install:
 - 'echo "this is a build for: $TRAVIS_BRANCH"'
 - 'if [[ "$TRAVIS_BRANCH" != "devel" ]]; then bash ./.travis/install_cplex.sh; fi'
 install:
-- pip install flake8 numpy scipy pyzmq pandas pytest pytest-cov
```

Comment:
I don't think it's necessary but that line was there before. I only pinned the pandas version.